packages/file-adapters/README.md

```diff
@@ -24,10 +24,11 @@ const fileAdapter = new LocalFileAdapter({
 
 ### Config
 
-| Option | Type     | Default        | Description                                                            |
-| ------ | -------- | -------------- | ---------------------------------------------------------------------- |
-| `src`  | `String` | Required       | The path where uploaded files will be stored on the server.            |
-| `path` | `String` | Value of `src` | The path from which requests for files will be served from the server. |
+| Option        | Type        | Default        | Description                                                                                                 |
+| ------------- | ----------- | -------------- | ----------------------------------------------------------------------------------------------------------- |
+| `src`         | `String`    | Required       | The path where uploaded files will be stored on the server.                                                 |
```

Comment:
Does this need a `yarn format` to avoid the diff on the `src` and `path` lines?

Reply:
The formatter did that to match the column widths.

Reply:
@timleslie the reason you see changes to multiple lines of a mark-down table when only one line is changed is that the cols are justified so other rows resize to match content. (I might be mansplaining, but just in case you didn't realise this)